package.json

```diff
@@ -11,17 +11,19 @@
   "bugs": "https://github.com/wayfair/tungstenjs/issues",
   "main": "tungsten.js",
   "scripts": {
+    "clean": "rm dist/*.js",
```

Comment:
I would like to introduce `clean` as the first step in a build.  This implementation however is not travis friendly (and breaks ci).  I can pull this out for now or leave it in for utility.